manager/integration/tests/backupstore.py

```diff
@@ -53,6 +54,7 @@ def set_random_backupstore(request, client):
         mount_nfs_backupstore(client)
 
     yield
+    cleanup_all_volumes(client)
```

Comment:
I know this is a workaround, but we should have a better way instead of adding non-backup clean up logic here?

Do we have any teardown place for each test case? probably we can add this there?

Reply:
Do you think we should have a consolidated place for all teardown logic?
AFAIK ATM we are depending on the fixture to clean up itself after each test case. This `set_random_backupstore` is a fixture so it seems to be ok to add it here.

Reply:
Probably no side effect but it's just more about the segregated responsibility for each fixture.

If mixing things (here cleanup/teardown) in different fixtures, u will probably don't know what teardown each fixture does.

However, I think let's merge this first to see how it works first then revisit this when reviewing all fixtures.